geniza/entities/admin.py

```diff
@@ -132,7 +172,13 @@ def get_form(self, request, obj=None, **kwargs):
         """For Person-Person autocomplete on the PersonAdmin form, keep track of own pk"""
         if obj:
             self.own_pk = obj.pk
-        return super().get_form(request, obj, **kwargs)
+
+        # remove add/change/delete buttons from "role" widget
+        form = super().get_form(request, obj, **kwargs)
+        form.base_fields["role"].widget.can_add_related = False
+        form.base_fields["role"].widget.can_change_related = False
+        form.base_fields["role"].widget.can_delete_related = False
+        return form
```

Comment:
would it be better to handle by user permissions, limiting which user groups have access to add/change/delete these?

Reply:
Good question, asked the team! I know originally their decision was to make this editable only from the admin list view, and not from the inlines, but I've asked if they'd prefer group-based permissions.

Reply:
Thanks for raising this, the team agrees group-based permissions are preferable!